oshi-core/src/main/java/oshi/driver/linux/proc/UpTime.java

```diff
@@ -30,7 +31,7 @@ public static double getSystemUptimeSeconds() {
                 // No space, error
                 return 0d;
             }
-            return Double.parseDouble(uptime.substring(0, spaceIndex));
+            return ParseUtil.parseLastDouble(uptime.substring(0, spaceIndex), 0);
```

Comment:
Since you're doing the default here you don't need to catch the `NumberFormatException` below.  But should use `parseDoubleOrDefault()`.

Also, the compiler probably handles it but I like to include the `d` when defining (double) 0.

Reply:
Done